### DISCUSSION TITLE
Welcome to animity Discussions!

### DISCUSSION THREAD

**Discussion by kl3jvi** [kl3jvi/animity#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Noorislam-XD** [comment]
•  login through my anime list (mal)
•  And video quality change options

**Reply by kl3jvi** [reply]
Hey, are you a developer ? If yes we can go through it together. If not I will plan a release with these features in upcoming months.

**Comment by ITADORI-15** [comment]
Ig u should remove those reviews....its kind of annoying 

There is this bug where if I am playing a video and click on home and reopen the app from recents the video stars from 0(beginning)

**Comment by GoluLiliput** [comment]
The app new update is constantly crashing when u open any anime title. It crashes before fetching the episode list.

**Comment by RashikFarhan** [comment]
Hello, I really like your app. Could you please give external video player support so that I can play from vlc...

**Reply by kl3jvi** [reply]
@RashikFarhan I will include that in the next release.

**Comment by Salaz8117** [comment]
where's the apk?
